GetIntoTeachingApi/Models/Crm/Validators/CandidateSchoolExperienceValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using FluentValidation;
+using GetIntoTeachingApi.Services;
+using GetIntoTeachingApi.Validators;
+
+namespace GetIntoTeachingApi.Models.Crm.Validators
+{
+    public class CandidateSchoolExperienceValidator : AbstractValidator<CandidateSchoolExperience>
+    {
+        private readonly IStore _store;
+
+        public CandidateSchoolExperienceValidator(IStore store)
+        {
+            _store = store;
+
+            RuleFor(se => se.SchoolUrn).MaximumLength(8);
```

Comment:
Will we get a validation error on `CandidateId` when its not specified given its non-nullable, or does it default to an empty guid of 0's? I see to remember having to make these `Guid?` and have a `NotNull()` rule for them to behave, but I may be misremembering

Reply:
You may be misremembering, I have copied off the ApplicationForm model and that doesn't do this